cmd/commands/git-source.go

```diff
@@ -163,32 +183,170 @@ func RunGitSourceCreate(ctx context.Context, opts *GitSourceCreateOptions) error
 	}
 
 	if len(fi) == 0 {
-		if err = createDemoWorkflowTemplate(gsFs, opts.runtimeName); err != nil {
-			return fmt.Errorf("failed to create demo workflowTemplate: %w", err)
+		err = CreateCronExamplePipeline(&GitSourceCronExampleOptions{
+			RuntimeName:         opts.RuntimeName,
+			GsCloneOpts:         opts.GsCloneOpts,
+			GsFs:                gsFs,
+			EventSourceFileName: opts.EventSourceFileName,
+			SensorFileName:      opts.SensorFileName,
+		})
+		if err != nil {
+			return fmt.Errorf("failed to create cron example pipeline. Error: %w", err)
 		}
 
-		commitMsg := fmt.Sprintf("Created demo workflow template in %s Directory", opts.gsCloneOpts.Path())
+		pOpts := &git.PushOptions{
+			CommitMsg: fmt.Sprintf("Created demo workflow template in %s Directory", opts.GsCloneOpts.Path()),
+		}
 
-		log.G(ctx).Info("Pushing a demo workflow-template to the new git-source repo")
-		if err := apu.PushWithMessage(ctx, gsRepo, commitMsg); err != nil {
-			return fmt.Errorf("failed to push demo workflow-template to git-source repo: %w", err)
+		_, err = gsRepo.Persist(ctx, pOpts)
```

Comment:
we don't use this anymore, please switch to `apu.PushWithMessage` as it has built it retry mechanism